### PR TITLE
updated reference to player api

### PR DESCRIPTION
**Pull request by ThrivingKings**

### CODE REVIEW

**Comment by possibilities** [comment]
Can we take the same approach as the educreations gadget (NOTE this one already listens for WebComponentsReady, just needs the webcomponentsjs dependency, etc).

**Comment by ThrivingKings** [comment]
sure, updated.

File: bower.json [hunk 1]
```diff
@@ -10,7 +10,8 @@
   "dependencies": {
     "jquery": "~2.1.1",
     "versal-player-api": "~0.3.3",
-    "versal-gadget-api": "~0.0.1",
-    "versal-component-runtime": "~0.2.4"
+    "versal-component-runtime": "~0.2.4",
```
Comment (possibilities) [inline bower.json]: You can drop versal-component-runtime also.

Comment (possibilities) [inline bower.json]: Means you can also lose

```
<script src="bower_components/eventEmitter/EventEmitter.js"></script>
<script src="bower_components/versal-component-runtime/dist/runtime.min.js"></script>
```

**Comment by ThrivingKings** [comment]
updated

File: versal.html [hunk 2]
```diff
@@ -8,7 +8,6 @@
   <div class="main-app"></div>
 
   <script src="bower_components/eventEmitter/EventEmitter.js"></script>
```
Comment (possibilities) [inline versal.html]: You don't need to include EventEmitter directly anymore.

**Comment by ThrivingKings** [comment]
yeah, hold on. it's not working now that I've removed those. I pushed up the last commit just based on your response without testing. now, the gadget isn't loading

**Comment by ThrivingKings** [comment]
there we go

**Pull request merged**